ocean_lib/data_provider/data_service_provider.py

```diff
@@ -57,7 +57,8 @@ def set_http_client(http_client: Session) -> None:
     @staticmethod
     @enforce_types
     def encrypt(
-        objects_to_encrypt: Union[list, str, bytes], encrypt_endpoint: str
+        objects_to_encrypt: Union[list, str, bytes],
+        provider_uri: Optional[str] = "http://172.15.0.4:8030",
```

Comment:
it should't fall back to 172... It should fall back to the provider_uri configured inside the config object.

Reply:
I have tried that, but it fails for `check_types`  function from enforce typing. It waits for a property instead of string.

Reply:
then don't fallback using this syntax. Use default None and replace it inside the function itself.